packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/AppConfiguration.kt

```diff
@@ -344,6 +356,7 @@ public interface AppConfiguration {
                 logger = logConfig,
                 appName = appName,
                 appVersion = appVersion,
+                bundleId = bundleId,
```

Comment:
It would be nice with a test that this actually has a value that isn't the default

Reply:
I added https://github.com/realm/realm-kotlin/pull/1411/files#diff-dee641912c3d91b744306c306cb373fef716a5ed25027363713a71dde04e506fR417 which gets the value from https://github.com/realm/realm-kotlin/pull/1411/files#diff-ad7d5c068deb652e9518763651bed6874ac941679d187d260bec712efec3cf27R122. This doesn't verify that the value is injected correctly from the gradle plugin, but at least shows that the compiler plugin is injecting the plugin option.